pages/trial.tsx

```diff
@@ -114,6 +114,9 @@ export default function Trial({ props }) {
         send_to: "send_to': 'AW-467110449/dWjCCL2x78ICELGU3t4B", // trialRequest conversion
         event_callback: () => {},
       });
+      globalThis.gtag("event", "trial-request", {
+        event_category: "conversion",
+      });
```

Comment:
The other gTag events have `event_callback`... Should that be added to these new events, or perhaps removed from the others?

Reply:
The callback functions are empty so I don't think they're actually doing anything?

Here's the reference in the gtag.js docs https://developers.google.com/analytics/devguides/collection/gtagjs/sending-data

Reply:
Suppose I can try removing them and see if we still get data for our ads? 😅